templates/prometheus.go

```diff
@@ -155,6 +155,10 @@ var PrometheusTemplate = promv1.Prometheus{
 						Regex:        getRelableRegex(alerts, metrics),
 						Action:       "keep",
 					},
+					{
+						Regex:  "^documentation$",
+						Action: "labeldrop",
```

Comment:
Note to self: Need to add `documentation` field as a label on in Ceph OCS rules instead of an annotation.

Reply:
Also need to test this out.